dcos/config.py

```diff
@@ -12,6 +12,25 @@
 logger = util.get_logger(__name__)
 
 
+def get_global_config():
+    """Returns the path to the deprecated global DCOS config file.
+
+    :returns: path to the DCOS config file
+    :rtype: str
+    """
+
+    return os.environ.get(constants.DCOS_CONFIG_ENV, get_default_config_path())
+
+
+def get_clusters_path():
```

Comment:
Why get_clusters_**path** and not config ? be consistent with get_global_**config**?